tests/h/search/query_test.py

```diff
@@ -74,6 +74,7 @@ def test_it_uses_new_filter_syntax_for_es6(self):
                                   "must": []}}
 
 
+@pytest.mark.usefixtures('pyramid_request')
```

Comment:
This isn't actually necessary because the `search` fixture is already parametrized so any tests which use it will automatically run with both ES versions.